certified-connectors/Gratavid/apiDefinition.swagger.json

```diff
@@ -0,0 +1,542 @@
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "label": {
+                        "type": "string",
+                        "description": "label",
+                        "title": "",
+                        "x-ms-visibility": "important"
+                      },
+                      "sample": {
+                        "type": "string",
+                        "description": "sample"
+                      },
+                      "value": {
+                        "type": "string",
+                        "description": "value"
+                      }
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Gratavid Data",
+        "description": "Get Gratavid notes and users",
+        "operationId": "getData",
+        "parameters": [
+          {
+            "name": "source",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "microsoftFlow",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "event",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "important",
+            "enum": [
+              "getUsers",
+              "getNotes"
+            ]
+          }
+        ],
+        "x-ms-visibility": "internal"
+      },
+      "delete": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Webhook Unsubscribe",
+        "description": "Webhook Unsubscribe",
+        "operationId": "webhookUnsubscribe",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "source",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "microsoftFlow",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "event",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "webhookUnsubscribe",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "webhookSubscribeId",
+            "in": "query",
+            "required": false,
+            "type": "integer"
+          },
+          {
+            "name": "webookHookEvent",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ]
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "New Event",
```

Comment:
@Wtrapp Usually summary values starts with "When.." or "Trigger..", Please update it accordingly.